azure/datalake/store/core.py

```diff
@@ -249,13 +259,20 @@ def _acl_call(self, action, path, acl_spec=None):
             '[default:]user|group|other:[entity id or UPN]:r|-w|-x|-,[default:]user|group|other:[entity id or UPN]:r|-w|-x|-,...'
 
             Note that for remove acl entries the permission (rwx) portion is not required.
+        invalidate_cache: bool
+            optionally indicates that the cache of files should be invalidated after this operation
+            This should always be done for set and remove operations, since the state of the file or folder has changed.
         """
         parms = {}
         path = AzureDLPath(path).trim()
         if acl_spec:
             parms['aclSpec'] = acl_spec
         
-        return self.azure.call(action, path.as_posix(), **parms)
+        to_return = self.azure.call(action, path.as_posix(), **parms)
```

Comment:
just confirm, `path.as_posix` is fast enough so you can call it several time without a variable?